packages/api/src/controllers/notification.ts

```diff
@@ -0,0 +1,107 @@
+import { sendgridEmail } from "./helpers";
+import { frontendUrl, unsubscribeUrl } from "./user";
+import { db } from "../store";
+import { sendgridEmailPaymentFailed } from "./helpers";
+import { WithID } from "../store/types";
+import { Request } from "express";
+import { User } from "../schema/types";
+
+export const getUsageNotifications = async (
+  usagePercentages,
+  user: WithID<User>
+) => {
+  let notifications = [];
+
+  if (
+    usagePercentages.TotalUsageMins > 90 ||
+    usagePercentages.DeliveryUsageMins > 90 ||
+    usagePercentages.StorageUsageMins > 90
```

Comment:
I think these fields have confusing names. They imply the unit is minutes, when I believe it is actually a percentage? WDYT of renaming to e.g. `TotalUsageLimitPercent`, `just `...Percent` or sth like that?